index.js

```diff
@@ -1 +1,110 @@
-// implement your API here
+const express = require('express');
+const db = require('./data/db.js');
+const cors = require('cors');
+
+const server = express();
+
+server.use(express.json());
+server.use(cors());
+
+const PORT = 5000;
+
+server.post('/api/users', (req, res) => {
+  const newUser = req.body;
+  db.insert(newUser)
+    .then(user => {
+      if (!user.name === '' || user.bio === '') {
```

Comment:
Implementing you check here is too late to prevent the database from inserting the data as empty string.